concent_api/core/management/commands/concent_migrate_ids_to_uuid.py

```diff
@@ -0,0 +1,25 @@
+from logging import getLogger
+
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+
+from core.constants import REGEX_FOR_VALID_UUID
+from core.models import StoredMessage
+from core.models import Subtask
+
+logger = getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = 'Migrates subtask_id and task_id to UUID standard'
+
+    def handle(self, *args, **options):
+        subtasks_deleted = Subtask.objects.exclude(
+            Q(task_id__iregex=REGEX_FOR_VALID_UUID) |
+            Q(subtask_id__iregex=REGEX_FOR_VALID_UUID)
+        ).delete()
+        stored_messages_deleted = StoredMessage.objects.exclude(
+            Q(task_id__iregex=REGEX_FOR_VALID_UUID) |
+            Q(subtask_id__iregex=REGEX_FOR_VALID_UUID)
+        ).delete()
+        logger.info(f'Migrations deleted {subtasks_deleted[0]} subtasks and {stored_messages_deleted[0]} stored messages')
```

Comment:
I would change this message to `Deleted ....`. The `Migrations` is misleading as it is not a migration.

Reply:
Done